src/Compilers/Core/Portable/Text/TextLineCollection.cs

```diff
@@ -62,6 +62,11 @@ public LinePositionSpan GetLinePositionSpan(TextSpan span)
         /// </summary>
         public int GetPosition(LinePosition position)
         {
+            if (position.Line >= this.Count)
+            {
+                throw new ArgumentOutOfRangeException(nameof(position.Line), string.Format(CodeAnalysisResources.LineCannotBeGreaterThanEnd, position.Line, this.Count));
```

Comment:
>string.Format(CodeAnalysisResources.LineCannotBeGreaterThanEnd, position.Line, this.Count) [](http://example.com/codeflow?start=77&length=90)

When `position.Line == this.Count`, it looks like the exception message will be something like "The requested line number 3 is greater than the number of lines 3."

Should the message be "The requested line number 3 must be less than ..." instead?